src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
Can you help me understand this a little better? Is it because with your other package update we are now pulling in Sockets where we didn't before? If so I'm still not sure why that would cause a conflict. 

cc @davidsh @CIPop